packages/tic-tac-toe/contracts/tests/types/app-data.ts

```diff
@@ -0,0 +1,142 @@
+// import {Weapon} from './weapons';
+import {defaultAbiCoder, bigNumberify} from 'ethers/utils';
+import {unreachable} from '../utils/unreachable';
+
+export enum PositionType {
+  Start, // 0
+  XPlaying, // 1
+  OPlaying, // 2
+  Draw, // 4
+  Victory // 5
+}
+
+export interface TTTData {
+  positionType: PositionType;
+  stake: string; // uint256
+  Xs: number; // uint16
+  Os: number; // uint16
+}
+export interface Start {
+  type: 'start';
+}
+export interface XPlaying {
+  type: 'xPlaying';
+  stake: string;
+  Xs: number;
+}
+
+export interface OPlaying {
+  type: 'oPlaying';
+  stake: string;
+  Os: number;
+}
```

Comment:
Do we not want these interfaces to include *both* `Xs` and `0s`? Otherwise, it seems as if they might just get filled with defaults on L76.